api-server/config/openapi.yml

```diff
@@ -113,7 +113,7 @@ paths:
       requestBody:
         required: true
         content:
-          application/x-www-form-urlencoded:
+          application/json:
```

Comment:
I've chose to change it to json because I had a bit of an issue with the parameter serialization between the code generated routine and the api-server. IIRC the request was correctly populated by `x-www-form-urlencoded ` variables, but the server couldn't recover them for some reason. Switching to `json` solved the issue.

It shouldn't pose any issue for the login form, as the values would be passed to the login hook as they are now done.

Reply:
See, when we call the mutation to login https://github.com/artemiscloud/activemq-artemis-self-provisioning-plugin/pull/130/files#diff-43a4379a03fa59a3cab4d52b04b5eaa1df6059c2107e28cc1db0d0e88fed3e53R397 we're passing the necessary values as a parameter.
If these would be gotten from a form it would mean that on `onsubmit` we would call the mutation passing the form values in the `requestBody` object.

Reply:
ok understand now. Thanks for explanation